osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -128,6 +135,9 @@ public override void OnEntering(IScreen last)
                     track.Seek(metadata.PreviewTime != -1 ? metadata.PreviewTime : 0.4f * track.Length);
                     track.Start();
                 }
+
+                if (api?.State == APIState.Offline)
```

Comment:
which is this inside a track-related conditional?

Reply:
didn't realize it was checking the track as well, I'll make a separate clause where it only checks if the last screen was an `IntroScreen`